settings/system/frontend.rst

```diff
@@ -1,8 +1,39 @@
 Frontend
 ********
 
-Send client stats: Default ``no`` (inactive)
+Send client stats
    Send client stats/error message to central server to improve the usability.
 
-Client storage: Default ``no`` (inactive)
+   Default: ``no`` (inactive)
+
+   .. danger::
+
+      This sends logging to our central logging server.
+      Log files of Zammad can contain *very sensitive* data like email addresses
+      and other. **Do not** enable this option unless we're asking you to.
+
+Client storage
    Use client storage to cache data to enhance performance of application.
+
+   Default: ``no`` (inactive)
+
+.. _core-workflow-ajax-mode:
+
+Core Workflow Ajax Modus
+   This setting allows administrators to enforce :doc:`/system/core-workflows`
+   to use Ajax-Calls instead of web sockets. You'll usually only need this if
+   you experience serious issues as noted below.
+
+      .. hint:: **🤓 Possible (technical) reasons**
+
+         In some cases your network structure (e.g. firewalls, proxies)
+         may disconnect long web socket connections. This leads to
+         select fields staying empty (e.g. owner selection *after* selecting
+         your group) or fields not shown / hidden (e.g. when switching to
+         or from pending states, the pending till field not showing / hiding).
```

Comment:
I would add colons: the "pending until" field not showing

Reply:
![image](https://user-images.githubusercontent.com/6549061/137118928-836edbdf-0bd3-4eb2-88ca-9d9b5a955264.png)

The field is actually called "pending till", but I'll add colons to help understanding that better.